datasource/management/commands/groot-352.py

```diff
@@ -0,0 +1,109 @@
+import re
+from datetime import datetime
+
+from django.core.exceptions import ValidationError
+from django.core.management.base import BaseCommand
+from django.core.validators import URLValidator
+from django.utils import timezone
+
+from brand.models import *
+from datasource.models import *
+
+
+class Command(BaseCommand):
+    help = "This script enters information from usnic into brands for FCU entity type"
+
+    def handle(self, *args, **kwargs):
+        print("Running db_changes_fcu.py")
+        rssd_ids = []
+        with open(r"datasource\management\commands\cdfi_rssd_numbers.txt", "r") as file:
+            rssd_ids = [line.strip() for line in file]
```

Comment:
You're using a list comprehension here which is nice. However, this data structure should be a set. Otherwise you're iterating through it len(filtered_usnic) x len(rssd_ids) times. 

This video may help you understand it: https://www.youtube.com/watch?v=xzDEEzH8XWk

Reply:
This is all that will take:

`rssd_ids = set([line.strip() for line in file])`